### ISSUE TITLE
Support kdf paths on Keystore

### ISSUE THREAD

**Issue by MegaRedHand** [lambdaclass/lambda_ethereum_consensus#1030]
Context: #1013

**Comment by afsanehForsat** [comment]
comment

**Comment by artemd24** [comment]
I'd like to tackle this!

**Comment by MegaRedHand** [comment]
Sure! Go ahead!

**Comment by artemd24** [comment]
Hello @MegaRedHand!
I have started this issue, but it's a bit unclear for me, so I have a question about implementation.
Should I make `password` parameter in `decode_str!` optional and in case of `nil` get "secret" from `decoded_json["path"]` file?

**Comment by MegaRedHand** [comment]
No. The _path_ attribute is for deriving the secret key. I'm not sure how it works, but once the secret is retrieved you should be able to use secret + path to derive a secret key. There is more info in the EIP linked in the issue.

**Comment by Arkenan** [comment]
Unassigned (more than 1 month of inactivity)
